src/status_im2/common/bottom_sheet/styles.cljs

```diff
@@ -21,9 +21,20 @@
    :right                   0
    :border-top-left-radius  20
    :border-top-right-radius 20
+   :overflow                :hidden
```

Comment:
This change caused bottom-sheet selected item to not show.
Fixed in https://github.com/status-im/status-mobile/pull/15677